src/app/services/utility.service.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UtilityService {
+
+  // constructor() {}
+  IMG_TEXT = "Image icon";
+  SHOW_SPLASH = true;
```

Comment:
Please use separate module for consts and import them wherever needed.

Reply:
I have declare a folder inside src directory. And stored, exported valued in a .ts file from that directory.